libcodechecker/libhandlers/server.py

```diff
@@ -342,6 +392,186 @@ def arg_match(options):
     parser.set_defaults(func=__handle)
 
 
+def print_prod_status(prod_status):
+    """
+    Print the database statuses for each of the products.
+    """
+
+    header = ['Product endpoint', 'Database status',
+              'Schema version in the database',
+              'Schema version in the package']
+    rows = []
+
+    for k, v in prod_status.items():
+        db_status, schema_ver, package_ver = v
```

Comment:
For me it would be better to see what product has the same schema version as the package:
```bash
----------------------------------------------------------------------------------------------------------------------------------------
Product endpoint | Database status                                      | Schema version in the database | Schema version in the package
----------------------------------------------------------------------------------------------------------------------------------------
Default          | Database is OK.                                      | 82ca43f05c10 (up to date)      | 82ca43f05c10                 
Test             | Database is OK.                                      | 82ca43f05c10 (up to date)      | 82ca43f05c10                 
Xerces           | Database schema mismatch migration is not available. | 8268fc7ca7f4                   | 82ca43f05c10                 
----------------------------------------------------------------------------------------------------------------------------------------
```